index.html

```diff
@@ -30,7 +30,18 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
     </div>
   </section> 
   <!--Секция 2-->
-  <section>Секция 2</section>
+  <section class="warranty" id="warranty">
+    <div class="warranty__wrapper">
+      <div class="warranty__text-block">
+        <h2 class="warranty__title">1-year warranty boost</h2>
+        <p class="warranty__copy">ZEN cards have a very neat feature that will help you whenever your electronic devices start failing after the standard warranty ends, which is pretty common.</p>
+        <p class="warranty__copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty gets extended by one year. How does it work? It simply does. Automatically.</p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
```

Comment:
It is recommended to close single tag at the end.
Like this  <img src="./assets/images/2.jpg" alt="Basket" />